stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -279,6 +279,14 @@ impl FromRow<LeaderBlockCommitOp> for LeaderBlockCommitOp {
 
         let burn_parent_modulus: u8 = row.get_unwrap("burn_parent_modulus");
 
+        let punished_str: Option<String> = row.get_unwrap("punished");
```

Comment:
Is there a more economical way to encode this?

Reply:
Of course, but I'm not convinced that its necessary.